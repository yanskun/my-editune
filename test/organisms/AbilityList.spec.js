import { createLocalVue, shallowMount } from "@vue/test-utils"
import ElementUI from "element-ui"
import Component from "~/components/organisms/AbilityList"

const localVue = createLocalVue()
localVue.use(ElementUI)

describe("AbilityList", () => {
  test("snapshot", () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        abilities: [{
          value: false,
          label: "test"
        },
        {
          value: 1,
          label: "test"
        }]
      },
      localVue
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
