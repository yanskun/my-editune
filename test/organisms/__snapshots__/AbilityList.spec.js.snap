// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AbilityList snapshot 1`] = `
<div
  class="abilityList"
>
  <div
    class="ability"
  >
    <switch-with-label-stub
      label="test"
    />
  </div>
  <div
    class="ability"
  >
    <input-with-label-stub
      label="test"
      type="number"
      value="1"
    />
  </div>
</div>
`;
